Makefile

```diff
@@ -1,21 +1,30 @@
 DEFAULT: build
 
-GO           ?= go
-GOFMT        ?= $(GO)fmt
-FIRST_GOPATH := $(firstword $(subst :, ,$(shell $(GO) env GOPATH)))
+GO             ?= go
+GOFMT          ?= $(GO)fmt
+FIRST_GOPATH   := $(firstword $(subst :, ,$(shell $(GO) env GOPATH)))
 TRICKSTER_MAIN := cmd/trickster
-DEP          := $(FIRST_GOPATH)/bin/dep
-TRICKSTER    := $(FIRST_GOPATH)/bin/trickster
+TRICKSTER      := $(FIRST_GOPATH)/bin/trickster
+PROGVER        := $(shell grep 'applicationVersion = ' $(TRICKSTER_MAIN)/main.go | awk '{print $$3}' | sed -e 's/\"//g')
+GO111MODULE    ?= on
+export GO111MODULE
 
-PROGVER = $(shell grep 'applicationVersion = ' $(TRICKSTER_MAIN)/main.go | awk '{print $$3}' | sed -e 's/\"//g')
 
 .PHONY: go-mod-vendor
 go-mod-vendor:
-	GO111MODULE=on $(GO) mod vendor
+	$(GO) mod vendor
+
+.PHONY: go-mod-tidy
+go-mod-tidy:
+	$(GO) mod tidy
+
+.PHONY: test-go-mod
+test-go-mod: go-mod-tidy go-mod-vendor
+	@git diff --quiet --exit-code go.mod go.sum || echo "There are changes to go.mod and go.sum which needs to be commited"
```

Comment:
TravisCI is failing because of the typo here on the word committed. If you can fix that it should pass all the checks. Thanks!